### ISSUE TITLE
Drop-down List

### ISSUE THREAD

**Issue by mauriziopozzoni** [istSOS/istsos2#5]
There are currently 3 drop-down lists: service, property and procedure.
Between property and procedure, there should be another one: observedProperty, in order to view in the last dropo-down list only the procedures with the desired property.

**Comment by mantonovic** [comment]
Hi @mauriziopozzoni ,
Are you speaking about the data editor / viewer? If this is the case, it could be really a nice improvement to filter procedures based on the observed properties.

![screenshot from 2016-09-22 11 50 01](https://cloud.githubusercontent.com/assets/1797931/18743742/bc26d20e-80ba-11e6-8d2a-07db41b38874.png)

**Comment by mauriziopozzoni** [comment]
Exactly. WIth more than one hundred procedures, it could be a very useful filter
